python/extensions/python.bzl

```diff
@@ -12,7 +12,34 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"Python toolchain module extensions for use with bzlmod"
+"""Python toolchain module extensions for use with bzlmod.
+
+## Basic usage
```

Comment:
I think the `##` headers are messing up the rendered docs -- in the rtd preview doesn't show "python" as a heading anymore.